dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
Well, if I added such a way it's because it's the way we currently do in RIOT. Tools like OpenSSL and so are not in RIOT for now, so the point here is to use what we have, and if a security bug is found on the tweetnacl library I think anyways "security" using this module will be broken anyways.

`dev/random` exists in Linux and OS X, but obviously not in Windows (AFAIK we don't support Windows, even unofficially).